apiserver/params/params.go

```diff
@@ -213,6 +213,7 @@ type AddMachineParams struct {
 	// is set.
 	InstanceId              instance.Id                      `json:"instance-id"`
 	Nonce                   string                           `json:"nonce"`
+	DisplayName             string                           `json:"display-name"`
```

Comment:
This doesn't appear to be used

Reply:
Have removed the field. Only manual providers such as ssh and winrm use the `InstanceId` and `Nonce` fields. And even then, using `DisplayName` here wouldn't be much use as the params are used to fill in `MachineTemplate`s. `MachineTemplate` does not have a `DisplayName` field.